lib/dialects/sqlite3/query/compiler.js

```diff
@@ -104,7 +111,50 @@ class QueryCompiler_SQLite3 extends QueryCompiler {
       }
       blocks[i] = block.join(', ');
     }
-    return sql + ' select ' + blocks.join(' union all select ');
+    sql += ' select ' + blocks.join(' union all select ');
+
+    const { onConflict, ignore, merge } = this.single;
+    if (onConflict && ignore) sql += ' where true' + this._ignore(onConflict);
```

Comment:
Could we setup prettier to force having braces also in one-liner `if` statements? 

Those are pretty error prone when code is modified later on and in merging errors (here is one pretty public example of it https://blog.codecentric.de/en/2014/02/curly-braces/ ). 

Also in this case where the first `if` doesn't have braces and the second `else if` does hurt my eyes 😅

Reply:
totally support the curly braces, regardless of the 1st of April post date :)